### PR TITLE
changed label format and spacing

### PR DESCRIPTION
**Pull request by RachellCalhoun**
subtitle Title of the page: made it two lines. I changed the second section label to 'teach and learn' I thought we can add more info later/link to the weekly meetups page once it's ready. I also put a space for workshop/hakathon it was bothering me on mobile.

### CODE REVIEW

File: landingsite/templates/landingsite/index.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
 <div class="android-be-together-section mdl-typography--text-center" style="background: url({% static 'landingsite/images/slide01.jpg' %})
 center 30% no-repeat;">
   <div class="android-font android-slogan">print ("Hello World")</div>
-  <div class="android-font android-sub-slogan">Welcome to Code for Everyone... be yourself. do your thing. see what's going on.</div>
+  <div class="android-font android-sub-slogan">Welcome to Code for Everyone... </div>
+  <div class="android-font android-sub-slogan">Be yourself. Do your thing. See what's going on.</div>
```
Comment (hassanabidpk) [inline landingsite/templates/landingsite/index.html]: We can change this too. It was in original template. I never changed it :fearful:

File: landingsite/templates/landingsite/index.html [hunk 2]
```diff
@@ -7,7 +7,8 @@
 <div class="android-be-together-section mdl-typography--text-center" style="background: url({% static 'landingsite/images/slide01.jpg' %})
 center 30% no-repeat;">
   <div class="android-font android-slogan">print ("Hello World")</div>
```
Comment (hassanabidpk) [inline landingsite/templates/landingsite/index.html]: We can do print ("Code For Everyone") ?

**Comment by RachellCalhoun** [comment]
Will do

**Comment by hassanabidpk** [comment]
Looks fine!

**Pull request merged**